### PR TITLE
Kevin tan/revamp landing

### PR DESCRIPTION
**Pull request by Kevin-Tan25**
Refactoring front end code with Tailwind, fixing minor padding errors to make website more clean

### CODE REVIEW

File: src/components/footer/Footer.jsx [hunk 1]
```diff
@@ -1,55 +1,97 @@
...
+    //             target='_blank'
+    //             href='https://www.youtube.com/channel/UCkcrUv-e7GndWtFdihI9FGQ/videos'
+    //           >
+    //             <BsYoutube className='fa_icons' />
+    //           </a>
+    //         </li>
+    //       </ul>
+    //     </div>
+    //   </footer>
+    // </div>
```
Comment (IcedTea2K) [inline src/components/footer/Footer.jsx:94]: I think this part is a bit crowded. Is this legacy code? If so do you think if it's better to pull it into archive or leave it here?

File: src/components/header/Header.jsx [hunk 2]
```diff
@@ -1,81 +1,115 @@
...
 function Header() {
   const [isActive, setIsActive] = useState(false);
+  useEffect(() => {
+    if (isActive) {
+      document.documentElement.classList.add('overflow-hidden');
+      document.body.classList.add('overflow-hidden');
+    } else {
+      document.documentElement.classList.remove('overflow-hidden');
+      document.body.classList.remove('overflow-hidden');
+    }
```
Comment (IcedTea2K) [inline src/components/header/Header.jsx:19]: This is such a cheeky walk-around. Me likey likey

**Review by IcedTea2K — APPROVED** [review]
Overall looks good. The new padding for the main page looks much cleaner now, but for the hero section and `who are we` seem a bit narrower than the rest of the page. We can always fix that later though.
image

File: src/components/footer/Footer.jsx (hunk 1 above)
Comment (Kevin-Tan25) [reply src/components/footer/Footer.jsx:94]: yeah yeah I agree I addressed the change in the what we offer section I'll open a MR up for it

**Pull request merged**